website/docs/cloud-docs/workspaces/settings/notifications.mdx

```diff
@@ -20,31 +20,36 @@ To add, edit, or delete notifications for a workspace, go to the workspace and c
 
 A notification configuration specifies a destination URL, a payload type, and the events that should generate a notification. To create a notification configuration:
 
-1. Click **Settings > Notifications**. The **Notifications** page appears.
-1. Click **Create a Notification**. The **Create a Notification** form appears.
-1. Configure the notifications:
+1.  Click **Settings > Notifications**. The **Notifications** page appears.
 
-   - **Destination:** Terraform Cloud can deliver either a generic payload or a payload formatted specifically for Slack, Microsoft Teams, or Email. Refer to [Notification Payloads](#notification-payloads) for details.
+2.  Click **Create a Notification**. The **Create a Notification** form appears.
 
-   - **Name:** A display name for this notification configuration.
+3.  Configure the notifications:
 
-   - **Webhook URL** This is only available for generic, Slack, and Microsoft Teams webhooks. The webhook URL is the destination for the webhook payload. This URL must accept HTTP or HTTPS `POST` requests and should be able to use the chosen payload type. For details, refer to the [Slack documentation - Create an Incoming Webhook](https://api.slack.com/messaging/webhooks#create_a_webhook) and the [Microsoft Teams documentation - Create an Incoming Webhook](https://docs.microsoft.com/en-us/microsoftteams/platform/webhooks-and-connectors/how-to/add-incoming-webhook).
-   - **Token** (Optional) This is only available for generic webhooks. A token is an arbitrary secret string that Terraform Cloud will use to sign its notification webhooks. Refer to [Notification Authenticity][inpage-hmac] for details. The token is encrypted for storage, so you cannot view it after saving the notification configuration.
+- **Destination:** Terraform Cloud can deliver either a generic payload or a payload formatted specifically for Slack, Microsoft Teams, or Email. Refer to [Notification Payloads](#notification-payloads) for details.
+- **Name:** A display name for this notification configuration.
+- **Webhook URL** This is only available for generic, Slack, and Microsoft Teams webhooks. The webhook URL is the destination for the webhook payload. This URL must accept HTTP or HTTPS `POST` requests and should be able to use the chosen payload type. For details, refer to the [Slack documentation - Create an Incoming Webhook](https://api.slack.com/messaging/webhooks#create_a_webhook) and the [Microsoft Teams documentation - Create an Incoming Webhook](https://docs.microsoft.com/en-us/microsoftteams/platform/webhooks-and-connectors/how-to/add-incoming-webhook).
+- **Token** (Optional) This is only available for generic webhooks. A token is an arbitrary secret string that Terraform Cloud will use to sign its notification webhooks. Refer to [Notification Authenticity][inpage-hmac] for details. The token is encrypted for storage, so you cannot view it after saving the notification configuration.
```

Comment:
```suggestion
- **Token** (Optional) This notification is only available for generic webhooks. A token is an arbitrary secret string that Terraform Cloud will use to sign its notification webhooks. Refer to [Notification Authenticity][inpage-hmac] for details. You cannot view the stored and encrypted token is encrypted after saving the notification configuration.
```

Suggest changing "This is" to "This notification is" to avoid "This" without a subject. Also suggest changing "The token is encrypted for storage, so you cannot view it after saving the notification configuration" to "You cannot view the stored and encrypted token is encrypted after saving the notification configuration" to avoid passive voice"

Reply:
hmm I see the passive voice. you know what? I think I'm going to shorten it to make it even better. THis is a good catch though!